lib/sort-related-items.js

```diff
@@ -26,6 +26,18 @@ module.exports = (related, type) => {
   return {
     relatedObjects: sortedRelated.object,
     relatedDocuments: sortedRelated.archive,
-    relatedChildren: sortedRelated.children
+    relatedChildren: sortedRelated.children.sort((a, b) => {
+      var aIdentifier = a.attributes.identifier;
+      var bIdentifier = b.attributes.identifier;
+
+      if (aIdentifier < bIdentifier) {
+        return -1;
+      }
+      if (aIdentifier > bIdentifier) {
+        return 1;
+      }
+
```

Comment:
The identifier is a string such as 'BAB/A', 'BAB/B', 'BAB/A/1', 'BAB/A/2', so they're being sorted alphanumerically. They should be unique, but there is a condition to return 0 if not, just in case.